### PR TITLE
Add a URL prefix if one is missing.

### LINKED ISSUE #88: Config loading doesn't handle servers w/o http(s)
When loading a config that looks like:

```yaml
apiVersion: v1
clusters:
- cluster:
    server: "192.168.1.135:8001"
  name: "nuc"
```

Connection requests fail, because there is no protocol for the URL.

However `kubectl` works properly. We should add a prefix if it is missing.

### PR DESCRIPTION
**Pull request by brendandburns**
Fixes #88

### CODE REVIEW

File: util/src/main/java/io/kubernetes/client/util/Config.java [hunk 1]
```diff
@@ -104,7 +104,15 @@ public static ApiClient fromConfig(Reader input) {
 
     public static ApiClient fromConfig(KubeConfig config) {
         ApiClient client = new ApiClient();
-        client.setBasePath(config.getServer());
+        String server = config.getServer();
+        if (!server.startsWith("http://") && !server.startsWith("https://")) {
+            if (server.indexOf(":443") != -1) {
```
Comment (lwander) [inline util/src/main/java/io/kubernetes/client/util/Config.java]: Should we rely on `config.verifySSL()` instead/as well?

Comment (brendandburns) [reply util/src/main/java/io/kubernetes/client/util/Config.java]: verifySSL is default true, so I don't think that's the right thing to test here...

Comment (lwander) [reply util/src/main/java/io/kubernetes/client/util/Config.java]: Ah good point.

**Comment by mbohlool** [comment]
LGTM

**Pull request merged**